backend/src/test/java/ca/bc/gov/hlth/mohums/controller/UsersControllerTest.java

```diff
@@ -55,5 +64,44 @@ public void testUserLocation_badGuid() {
         // If the Location header doesn't match what we expect, namely /users/VALID_GUID, don't touch it.
         assertEquals("https://common-logon-dev.hlth.gov.bc.ca/auth/admin/realms/moh_applications/users/NOT_A_GUID", newHeaders.getLocation().toASCIIString());
     }
+    
+     //PLR 	   : dc7b9502-3ffa-4ff8-be2e-ebfebe650590
+    // BCER-CP : 9ee2f550-307a-414a-bda4-d9c44235a2a4
```

Comment:
Use named constants for these values and then you won't need comments because it will be "self documenting".